php/libraries/LorisForm.class.inc

```diff
@@ -1180,7 +1180,7 @@ class LorisForm
 
                 if (isset($el['required'])
                     && $el['required'] === true
-                    && empty($value)
+                    && (is_null($value) || (empty($value) && $value !=0))
```

Comment:
It can be this because `empty(0)` evaluates to true even though it should be false